Configuration/PyReleaseValidation/test/test_mc_setup/test-MC_23_setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#The conditions will be changed to be the proper GT from ALCA including 
+#the BS smearing record from GT, for the moment we use the automatic one pre BPix hole
+#that in CMSSW_13_0_17 is 130X_mcRun3_2023_realistic_postBPix_v1
+
+echo "######## TEST MC 2023 Setup"
+echo ""
+
+$CMSSW_BASE/src/Configuration/PyReleaseValidation/test/test_mc_setup/test_MC_setup.sh \
+140X_mcRun3_2023_realistic_v3 Run3_2023 2023v12 CMSSW_13_0_17 \
```

Comment:
Can this use 13_0_18? That includes https://github.com/cms-sw/cmsdist/pull/9017 that should get the job beyond
```
----- Begin Fatal Exception 29-Mar-2024 17:11:50 CET-----------------------
An exception of category 'FileOpenError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing input source of type PoolSource
   [2] Calling RootInputFileSequence::initTheFile()
   Additional Info:
      [a] Input file file:step2.root could not be opened.
      [b] Fatal Root Error: @SUB=TList::Clear
A list is accessing an object (0x14b339b8b740) already deleted (list name = TList)

----- End Fatal Exception -------------------------------------------------
```
and demonstrate the next failure (that we are going to address)

Reply:
Sure, let me fix it.